python/requirements/test-requirements.txt

```diff
@@ -26,6 +26,8 @@ feather-format==0.4.1
 flask==2.1.3
 freezegun==1.1.0
 google-api-python-client==2.65.0
+google-cloud-bigquery=3.11.0
+google-cloud-bigquery-storage=2.20.0
```

Comment:
Could you move these to `data-test-requirements`?

Reply:
Done